packages/components/src/DraggableItemList.tsx

```diff
@@ -275,10 +277,11 @@ class DraggableItemList<T> extends PureComponent<
             'draggable-item-list-dragging-item-container',
             draggingItemClassName
           )}
-          // eslint-disable-next-line react/jsx-props-no-spreading
+          // eslint-disable-next-line react/jsx-props-no-spreading, react/destructuring-assignment
           {...provided.draggableProps}
-          // eslint-disable-next-line react/jsx-props-no-spreading
+          // eslint-disable-next-line react/jsx-props-no-spreading, react/destructuring-assignment
           {...provided.dragHandleProps}
+          // eslint-disable-next-line react/destructuring-assignment
```

Comment:
I don't think destructuring is needed on the next line?

Reply:
Ah duh. Well why not do what it's asking and just destructure all of these above, e.g.:
```
      const { innerRef, dragHandleProps, draggableProps } = provided;
      return (
        <div
          className={classNames(
            'draggable-item-list-dragging-item-container',
            draggingItemClassName
          )}
          // eslint-disable-next-line react/jsx-props-no-spreading
          {...draggableProps}
          // eslint-disable-next-line react/jsx-props-no-spreading
          {...dragHandleProps}
          ref={innerRef}
        >
```